api/envoy/extensions/filters/http/cache/v3alpha/cache.proto

```diff
@@ -80,5 +80,5 @@ message CacheConfig {
   //
   // Max body size the cache filter will insert into a cache. 0 means unlimited (though the cache
   // storage implementation may have its own limit beyond which it will reject insertions).
-  uint32 max_body_bytes = 4;
+  uint64 max_body_bytes = 4;
```

Comment:
I'm wondering if we need to deprecate this field and make a new one with the different type. @htuch ?

Reply:
I think the question is do we realistically expect to have cache objects > 4GB? If yes, then let's do this new/old deprecation dance, otherwise let's handle the type mismatch in the implementation.

Reply:
I don't know why anyone would want to set a size limit that's higher than 4G, but I have no objection to allowing it. I also have no objection to leaving it as is.